owncloudApp/src/main/java/com/owncloud/android/presentation/security/passcode/PassCodeActivity.kt

```diff
@@ -194,7 +194,24 @@ class PassCodeActivity : AppCompatActivity(), NumberKeyboardListener, EnableBiom
     private fun setTextListeners() {
         val numberOfPasscodeDigits = (passCodeViewModel.getPassCode()?.length ?: passCodeViewModel.getNumberOfPassCodeDigits())
         for (i in 0 until numberOfPasscodeDigits) {
-            passCodeEditTexts[i]?.setOnClickListener { hideSoftKeyboard() }
+
+            passCodeEditTexts[i]?.apply {
+                hideSoftKeyboard()
+
+                setOnKeyListener { v, keyCode, event ->
+                    if (keyCode == KeyEvent.KEYCODE_TAB && event.action == KeyEvent.ACTION_DOWN) {
+                        val focusDirection = if (passCodeEditTexts.indexOf(v) == 0) {
+                            View.FOCUS_DOWN
+                        } else {
+                            View.FOCUS_LEFT
```

Comment:
Why `FOCUS_LEFT` here? Wouldn't it be to the right?

Reply:
Yes, in the beginning I set to `FOCUS_LEFT`, because the navigation it wasn't working well with `FOCUS_RIGHT`. I don't know why. But with the `focus` to `false` it isn't affected. So i will change to the correct one.

Reply:
But what are these lines used for? If those `EditText`s are not focusable anymore, the `setOnKeyListener`s will never run, right?

Reply:
But what are these lines used for? If those `EditText`s are not focusable anymore, the `setOnKeyListener`s will never run, right?

Reply:
Yeah, it is not useful, deleted. Also the `onFocusChangeListener` have been deleted